tox.ini

```diff
@@ -97,6 +97,36 @@ commands =
     --cov-fail-under={env:STORAGE_COVERAGE:68} \
     {posargs:storage}
 
+[testenv:storage-user]
+passenv = {[base]passenv}
+setenv =
+    {[base]setenv}
+    COVERAGE_FILE=.coverage-storage
```

Comment:
We alway name the coverage report after the test name, so `.coverage-storage-user`